app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'no-reply-preassembly@.stanford.edu'
```

Comment:
Note: not using a valid address here may affect how spam filters categorize our emails.

Reply:
I was wondering about this as well.  (ugh - do we want a new email list for this?)

Reply:
I would leave this up to Ben when he is back.  I've used to fake email address on Revs without issue, but perhaps a mailing list makes sense here.